images/router/haproxy/conf/haproxy-config.template

```diff
@@ -58,7 +58,7 @@ listen stats :1936
 {{ end }}
 
 frontend public
-  bind :80
+  bind :{{getEnv "ROUTER_SERVICE_HTTP_PORT" "80"}}
```

Comment:
Could set those 2 fields specifically from the environment in the template data if this is going to have issues. Only did it via the env as I figured it could be useful later on.